frontend/eda/event-streams/EventStreams.tsx

```diff
@@ -24,16 +24,22 @@ export function EventStreams() {
   const rowActions = useEventStreamActions(view);
   return (
     <PageLayout>
-      <PageHeader title={t('Event Streams')} description={t('Event streams.')} />
+      <PageHeader
+        title={t('Event Streams')}
+        description={t(
+          'Event streams represent server side webhooks which ease the routing issues related to running webhooks ' +
+            'individually in a container or a pod. Sources can be swapped in a rulebook with a matching event stream.'
+        )}
+      />
       <PageTable
-        id="eda-event-streams-table"
```

Comment:
the id here should be `eda-event-streams-table` instead of `eda-webhooks-table` ??

Reply:
Will change it in a follow up PR